admin/acceptance.py

```diff
@@ -161,16 +166,19 @@ def start_nodes(self):
         box_version = vagrant_version(self.package_source.version)
         # Boot the VMs
         check_safe_call(
-            ['vagrant', 'up'],
+            ['vagrant', 'up', '--parallel'],
```

Comment:
I was excited about this change, but when I tested it locally, it didn't seem to make much of a difference. I'm not sure if the virtualbox provider supports parallel provisioning:
 * https://www.mail-archive.com/vagrant-up@googlegroups.com/msg00662.html

If you think it should work, then this might be a change that could be split into a separate issue / branch so that we can see the results in isolation from the other changes in the branch.